src/applications/representative-appoint/config/form.js

```diff
@@ -183,11 +182,11 @@ const formConfig = {
         claimantPersonalInformation: {
           path: 'claimant-personal-information',
           depends: formData => !preparerIsVeteran({ formData }),
-          initialData:
-            /* istanbul ignore next */
-            !!mockData && environment.isLocalhost() && !window.Cypress
-              ? mockData
-              : undefined,
+          // initialData:
```

Comment:
This only impacts local development and can easily be uncommented to enable data seeding. It interfered with prefill.